install/Dockerfile

```diff
@@ -1,36 +1,65 @@
 FROM ubuntu:23.04
 
 WORKDIR /workspace
+# Determine platform and architecture
+RUN ARCH=$(uname -m) && \
```

Comment:
Slight preference for a `get-tools.sh` script for all tool-fetching bits and a call to `RUN get-tools.sh` in Dockerfile. The get-tools.sh script looks like it would be usable outside of the container, and it would keep the Dockerfile a little more organized.

Reply:
Yeah can do